content/en/tracing/code_origins/_index.md

```diff
@@ -115,12 +115,11 @@ export DD_CODE_ORIGIN_FOR_SPANS_ENABLED=true
 
 - To search for all spans that include Code Origins, use the query `@_dd.code_origin.type:*` in the [APM Trace Explorer][1].
 
-### Code preview is not visible
-
-- Ensure that all [Source Code Integration][7] setup requirements are met.
-- For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews do not work. See the Node.js [Source Code Integration][9] documentation for more details. 
-
+### Code preview is not visible or the file is not found
 
+- Ensure that all [Source Code Integration][7] setup requirements are met, and that your `DD_GIT_*` tags are configured with the correct values.
```

Comment:
I'm not sure about this change:
1. We're linking to the SCI page, which describes how to configure `DD_GIT_*`, but then we also mention it inline here with an "and", implying that it's something the user needs to do _besides_ what they are instructed to do in the SCI setup requirements. If we want to include this, we at least should replace "and" with something like "specifically" or "including".
2. If we decide to include it, I don't like that we call it "`DD_GIT_*` tags". IMO it should either be "`DD_GIT_*` environment variables", or it should be something like "Git information is embedded correctly in your build artifacts" to make it more generic like it is in the SCI setup requirements section.